src/window_type.h

```diff
@@ -25,7 +25,6 @@ enum WindowNumberEnum {
 	WN_CONFIRM_POPUP_QUERY_BOOTSTRAP, ///< Query popup confirm for bootstrap.
 
 	WN_NETWORK_WINDOW_GAME = 0,     ///< Network game window.
-	WN_NETWORK_WINDOW_LOBBY,        ///< Network lobby window.
```

Comment:
The removal of a window number enum or a set of widgets technically breaks the GSWindow API. Though practically the GS can't be running when the lobby window is open, so... I guess it's not an issue. Also wonderful that the join/download window numbers are exposed to GSes, which will never be able to highlight (or close) them.

Reply:
We always told GS authors to use the enums supplied, and not the internal values.